web/angular-wallet/src/locales/bg.json

```diff
@@ -716,6 +716,7 @@
   "seeded": "Наличен",
   "select_color_scheme": "Изберете цветова схема",
   "select_contact_in_sidebar": "Моля изберете контакт в лявата лента.",
+  "select_node": "Select Node",
```

Comment:
i18n manager is capable to translate the things automatically using google translate API. I have an API  key and can do that